samples/AnomalyDetector/src/Generated/Models/AlignPolicy.Serialization.cs

```diff
@@ -58,7 +58,7 @@ protected virtual void JsonModelWriteCore(Utf8JsonWriter writer, ModelReaderWrit
 #if NET6_0_OR_GREATER
 				writer.WriteRawValue(item.Value);
 #else
-                    using (JsonDocument document = JsonDocument.Parse(item.Value))
+                    using (JsonDocument document = JsonDocument.Parse(item.Value, new JsonDocumentOptions { MaxDepth = 256 }))
```

Comment:
could we add a `static readonly` member of this JsonDocumentOptions in the `ModelSerializationExtensions` class so that here we could use it as a reference?

Reply:
sounds good

Reply:
updated